pkgs/desktops/gnome-3/misc/gnome-tweaks/default.nix

```diff
@@ -5,13 +5,13 @@
 
 let
   pname = "gnome-tweaks";
```

Comment:
How does the need for `sound-theme-freedesktop` affect us?

https://gitlab.gnome.org/GNOME/gnome-tweaks/blob/6edc0dee262618ba2b0c6b46904f19a5e386351b/NEWS#L52

P.S. I have some themes here: https://github.com/jtojnar/nixpkgs/tree/sound-theme

Reply:
I'm pretty sure that it would have to be in `buildInputs`, because they're looked up in `$XDG_DATA_DIRS/sounds`. 
But I don't think gnome-tweaks needs this directly?

If that's so, then maybe it would need to be in `environment.systemPackages`.

Reply:
Found the issue https://gitlab.gnome.org/GNOME/gnome-tweaks/issues/170. I wonder why it is not crashing for me even though I do not have the sound theme installed.

Reply:
Looks like they detect things properly
https://gitlab.gnome.org/GNOME/gnome-tweaks/blob/master/gtweak/tweaks/tweak_group_appearance.py#L38
https://gitlab.gnome.org/GNOME/gnome-tweaks/blob/master/gtweak/utils.py#L114

Don't think we'll have a problem putting it in `buildInputs`.

Reply:
It does not seem to be necessary then.